.github/workflows/ci.yaml

```diff
@@ -2,8 +2,6 @@
 name: CI
 
 on:
-  schedule:
-    - cron: '44 4 */2 * *'
```

Comment:
I would like to keep those. It is useful to see the state without a PR under test to compare the results with.

Reply:
Please don't take this wrongly, but does that mean that you will act upon CI failures then? I see no value in a periodically running CI just to have it. And currently it is just running, sending out failure emails and **no one** has the time to fix them.

Reply:
I have added them back